pages/docs/deployment/migrate-v2-to-v3.mdx

```diff
@@ -0,0 +1,223 @@
+- Improved integrations: Since the first major version, we built many new ways to integrate your code with Langfuse such as [Decorators](https://langfuse.com/docs/sdk/python/decorators) for Python. We would recommend to check out our [quickstart](https://langfuse.com/docs/get-started) to see whether there is a more convenient integration available for you.
+
+**Background**: Langfuse V3 relies on an event driven backend architecture.
+This means, that we acknowledge HTTP requests from the SDKs, queue the HTTP bodies in the backend, and process them asynchronously.
+This allows us to scale the backend more easily and handle more requests without overloading the database.
+The SDKs below 2.0.0 send the events to our server and expect a synchronous response containing the database representation of the event.
+If you rely on this data and access it in the code, your SDK will break as of Nov. 11th, 2024 for the cloud version and post-upgrade for self-hosted versions.
+
+### API Changes
+
+#### POST /api/public/ingestion
+
+The `/api/public/ingestion` endpoint is now asynchronous.
+It will accept all events as they come in and queue them for processing before returning a 207 status code.
+This means that events will _not_ be available immediately after acceptance by the backend and instead will appear
+eventually in subsequent read requests.
+
+The individual events accepted a `metadata` property within their body of type `string | string[] | Record<string, unknown>`.
+Only the `Record` type is supported within our UI and endpoints to perform queries and filter events.
+Therefore, we enforce an object type for `metadata` going forward.
+All incoming events with `{ metadata: string | string[] }` will have their metadata mapped to an object with key `metadata`,
+i.e. `{ event: { body: { metadata: "test" } } }` will be transformed to `{ event: { body: { metadata: { metadata: "test" } } } }`.
+
+#### POST /api/public/scores
+
+The `/api/public/scores` endpoint is now asynchronous.
+It behaves exactly as the `/api/public/ingestion` endpoint, but will return a 200 status code with a body of `{ id: string }` type.
+Before, the endpoint returned the created score object directly.
+This change is inline with our [API reference](https://api.reference.langfuse.com/#post-/api/public/scores) and therefore not considered breaking.
+
+#### Deprecated endpoints
+
+The following endpoints are deprecated since our V2 release.
+We continue to accept requests to these endpoints and will remove them in a future release.
+Their API behaviour changes to be asynchronous and the endpoints will only return the id of the created object.
```

Comment:
```suggestion
Their API behavior changes to be asynchronous and the endpoints will only return the id of the created object instead of the full updated record.
```

sth like this or similar to make clear what the difference is in this change

behavior (AE)